packages/.eslintrc.json

```diff
@@ -21,6 +30,14 @@
         "yoda": "error",
         "radix": "error",
         "eqeqeq": ["error", "always"],
+        "import/no-deprecated": "warn",
+        "lodash/import-scope": ["error", "member"],
```

Comment:
Rule conflict. We use `lodash-united` instead of `lodash` or `lodash-es` so this rule cannot detect when we're using lodash.